guides/plugins/plugins/storefront/add-scss-variables-via-subscriber.md

```diff
@@ -0,0 +1,238 @@
+# Add SCSS variables
```

Comment:
Please add this file to the global `SUMMARY.md` - otherwise it won't show up in the navigation.

Reply:
added